libs/model/src/index.ts

```diff
@@ -19,3 +19,5 @@ export * from './database';
 export * from './globalActivityCache';
 export * from './models';
 export * from './utils';
+
+export * from './policies';
```

Comment:
move this up to the public section, anything under //Internals should be eventually made private